.github/workflows/slow.yaml

```
@@ -14,6 +14,12 @@ on:
   push:
     branches: [ "auto" ]
 
+concurrency:
+  # Cancel ongoing tests in case of push to the same PR or staging branch,
+  # but let previous master commit tests complete.
+  group: ${{ github.workflow }}-${{ github.event.pull_request.number || github.ref }}
+  cancel-in-progress: ${{ github.ref != 'refs/heads/master' }}
```

This (copied) YAML does not match _this_ workflow: This workflow does not test PR commits and does not test master branch commits. It only tests "auto" branch commits.

```suggestion
  # Cancel ongoing tests in case of push to staging branch.
  group: ${{ github.workflow }}-${{ github.ref }}
  cancel-in-progress: true
```